File: 2148-minimum-number-of-moves-to-seat-everyone/minimum-number-of-moves-to-seat-everyone.dart
class Solution {
  int minMovesToSeat(List<int> seats, List<int> students) {
    seats.sort();
    students.sort();
    int n = seats.length;
    int ans = 0;
    for(int i = 0; i < n; i++){
        ans += (seats[i] - students[i]).abs();
    }
    return ans;
  }
}

File: 2148-minimum-number-of-moves-to-seat-everyone/minimum-number-of-moves-to-seat-everyone.cpp
class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
        int ans = 0;
        int n = seats.size();
        sort(seats.begin(), seats.end());
        sort(students.begin(), students.end());
        for(int i = 0; i < n; i++) {
            ans += abs(seats[i] - students[i]);
        }
        return ans;
    }
};